### DISCUSSION TITLE
chrome "new style rule" bugging out because of the unescaped double dot :

### DISCUSSION THREAD

**Discussion by torian257x** [tailwindlabs/tailwindcss#2433] [Help]
Hi guys

I used to do this all the time and it's sooo annoying with tailwind.
in chrome
![image](https://user-images.githubusercontent.com/2072058/93963217-33eb5e80-fd22-11ea-94b4-378a011bcffd.png)

E.g. I select an element, then press "New Style Rule" to quickly prototype an idea

but:

as tailwind has the : in the classes, it adds the wrong default class. What do I mean?

after I press "new style rule" it will for example add:
```
.flex.justify-center.align-middle.pt-6.sm:pt-40{
}
```

for me to prototype. See the error?

must be

```
.flex.justify-center.align-middle.pt-6.sm\:pt-40
```

as a result, the new style selector added by chrome is wrong, and I cannot add my prototyped styles.

is there something to fix that, how are you guys doing it?

**Comment by paulpopus** [comment]
For quick testing, wouldn't necessarily call this prototyping, I would do one of the following:

**Add classes to the markup**, you can just double click the class in the inspector to edit it and add a new one.
![Screenshot from 2020-09-23 12-39-59](https://user-images.githubusercontent.com/35137243/94007829-f0343b80-fd99-11ea-90d0-0b20c594a762.png)

**Add a new class entirely** by clicking on the `.cls` button in the Styles tab
![Screenshot from 2020-09-23 12-37-01](https://user-images.githubusercontent.com/35137243/94007573-8d42a480-fd99-11ea-89a6-9c7a0dc03736.png)
What this will do is show you a list of all classes on that element and actually suggest you classes that have already been applied elsewhere in the document, making it great for tailwind.

If I want to just add CSS instead, I simply add my custom rules to the `element.style` for inline styling as that will avoid any inheritance issues and apply style changes to the element directly.
![Screenshot from 2020-09-23 12-39-01](https://user-images.githubusercontent.com/35137243/94007763-d8f54e00-fd99-11ea-8532-d1fecf23e4dc.png)

**Reply by torian257x** [reply]
thanks, but that doesnt solve the problem.

adding it to style means now you have to hand hold your styles, because it's not in the inspector file to copy it out.
Same with adding to class attribute. Useless to me if I have to remember _everything_ I prepared.
